translation/zh-CN/ibc/1_IBC_TERMINOLOGY.md

```diff
@@ -0,0 +1,154 @@
+
+**有关一组示例用例，请参见[此处](./4_IBC_USECASES.md) 。**
+
+**有关设计模式的讨论，请参见[此处](./5_IBC_DESIGN_PATTERNS.md) 。**
+
+本文档以简明的英语提供了整个IBC规范集中使用的关键术语的定义。
+
+## 抽象定义
+
+### 参与者
+
+*参与者*或*用户* （可互换使用）是使用IBC协议进行交互的实体。参与者可以是一个人类最终用户，在区块链上运行的模块或智能合约，或者是能够签名交易的链下中继器进程。
+
+### 机器/链/账本
+
+*机器* ， *链* ， *区块链*或*账本* （可互换使用）是一种状态机（可以是一个分布式账本或“区块链”，但并不需要是严格意义上由块组成的链），它实现了部分或全部的IBC规范。
+
+### 中继器进程
+
+中继器*进程*是一个链下进程，负责通过扫描两个或更多机器的状态并提交交易来在机器之间中继IBC数据包和元数据。
+
+### 状态机
+
+特定链的*状态机*定义状态的数据结构以及一组规则，这些规则决定了合法的交易可以在基于链的共识算法所达成一致的当前状态上触发状态转换。
+
+### 共识
+
+*共识*算法是一组操作分布式账本的进程所使用的协议，通常用来在存在一定数量的拜占庭错误的情况下就同一状态达成一致。
+
+### 共识状态
+
+*共识状态*是一组关于共识算法状态的信息集，它可以用来验证该共识算法输出的结果（例如，签名区块头中的加密承诺根）。
+
+### 加密承诺
+
+加密*承诺*是一种可以廉价的验证成员是否存在于键/值映射的方法，映射中可以附带提交一个简短的见证字符串。
+
+### 区块头
+
+*区块头*是对特定区块链共识状态的更新，包含当前状态的加密承诺，并可以通过“轻客户端”算法以明确定义的方式进行验证。
+
+### 加密承诺证明
+
+*加密承诺证明*是一种证明结构，用于证明特定键是否映射到加密承诺集中的特定值。
+
+### 处理模块
+
+IBC *处理模块*是状态机中实现了[ICS 25](../spec/ics-025-handler-module)的模块 ，它管理客户端，连接和通道，验证证明并存储数据包的加密承诺。
+
+### 路由模块
+
+IBC *路由模块*是状态机中实现了[ICS 26](../spec/ics-026-routing-module)的模块 ，它在使用路由模块的外部接口的主机状态机内路由处理模块和其他模块之间的数据包。
+
+### 数据报
+
+*数据报*是在某个物理网络上传输的不透明字节串，账本状态机中实现的IBC路由模块负责对其处理。在一些实现中，数据报可以是账本的交易或消息结构中的一个特定字段，并且可以包含其他信息（例如，防止垃圾消息的费用，防止重放的随机数，路由到IBC处理程序所需要的类型标识符等）。 所有IBC子协议（例如，创建连接，创建通道，发送数据包）都是由一组数据报和协议来定义的，以便于路由模块处理。
+
+### 连接
+
+*连接*是两个链上的一组持久化的数据结构，其中包含连接中关于另一个账本的共识状态信息。更新一个链的共识状态会改变对应的另一个链上的连接对象的状态。
+
+### 通道
+
+*通道*是两条链上的一组持久化的数据结构，其中包含方便数据包排序，仅一次送达和防止重放所需的元数据。通过通道发送数据包会更改其内部状态。通道与连接具有多对一关系，也就是说，单个连接可以具有任意数量的关联通道，所有通道必须关联着某个连接，该关联连接必须在创建通道之前创建。
+
+### 数据包
+
+*数据包*是一种特定的数据结构，它包含与序列相关的元数据（由IBC规范定义）和被称为数据包*数据* （具有由应用层定义的语义，例如通证数量和面额）的不透明数据段。数据包通过特定的通道发送（进一步说，通过特定的连接发送）。
+
+### 模块
+
+*模块*是特定区块链状态机的一个子组件，可以与IBC处理程序交互并根据发送或接收的特定IBC数据包的*数据*字段更改状态（例如，铸造或销毁通证）。
+
+### 握手
+
+*握手*是一类涉及多种数据报的子协议，通常用于初始化两个链上的某些公共状态，例如双方共识算法的信任状态。
+
+### 子协议
+
+子协议定义了一组数据报类型和函数，IBC处理模块必须实现这组定义来实现区块链。
+
+数据报必须通过外部中继器进程在链之间进行中继。中继器进程可以以任意方式运行-安全保证不应受其行为的影响，尽管整个流程通常需要存在至少一个忠实的中继器进程。
+
+以在两条链`A`和`B`之间的交互来探讨IBC子协议-这两条链之间没有先验区别，并且假定它们正在执行相同的，正确的IBC协议。`A`通常是在子协议过程中先行的那条链，而`B`次之。定义协议时通常应避免在变量名中包含`A`和`B`以避免混淆（因为链本身不知道它们在协议中是`A`还是`B` ）。
+
+### 身份认证
+
+*身份认证*是一个属性，用来确保数据报实际上是由特定链以IBC处理模块定义的方式发送的。
+
+## 属性定义
+
+### 最终性
+
+*最终性*是由共识算法提供的可量化保证，即处于某种验证人集合的特定假设下，特定区块不会回滚。 IBC协议需要最终性，尽管它不一定是绝对的（例如，一种针对中本聪共识算法的阈值最终性小工具，可以根据矿工行为的经济假设提供最终性）。
+
+### 不良行为
+
+*不良行为*是由共识算法定义的一类共识错误，并可由该共识算法的轻客户端检测（也可能是引发）。
+
+### 分歧错误
+
+*分歧错误*是一类特殊的共识错误，它由一个或多个验证人以非法的方式对单个块的多个不同后继块进行了投票造成。
+所有的分歧错误都属于不良行为。
+
+### 数据可用性
+
+*数据可用性*是指链下中继器进程在一定时间范围内可以从机器中获取状态数据的能力。
+
+### 数据机密性
+
+*数据机密性*是主机状态机可以拒绝向特定方提供特定数据而不损害IBC协议功能的能力。
+
+### 不可否认性
+
+*不可否认性*是指机器无法对已发送的特定数据包或提交的特定状态成功的进行质疑。 抛开状态机对于数据机密性的让步，IBC属于不可否认的协议。
+
+### 共识活性
+
+*共识活性*是指特定机器在共识算法下可以持续的产生区块。
+
+### 交易活性
+
+*交易活性*是指特定机器在共识算法下对传入交易（该交易应根据上下文明确）可以持续确认。交易活性依赖共识活性，但是共识活性不一定能提供交易活性。同时交易活性意味着抗审查能力。
+
+### 有界共识活性
+
+*有界共识活性*是指在特定范围内拥有共识活性。
+
+### 有界交易活性
+
+*有界交易活性*是指在特定范围内拥有交易活性。
+
+### 仅一次保证
+
+*仅一次保证*是指一个数据包仅确认不超过一次（通常仅一次保证意味着最终交易活性）。
+
+### 送达或超时保证
+
+*送达或超时保证*是指数据包或者被送达并执行，或者以给发件人回传响应的方式超时。
```

Comment:
A suggestion：

> *送达或超时保证*是指数据包发送者获得数据包已被送达并执行，或数据包已超时最终结果的特性。

Reply:
Thanks for your review. The [latest version](https://github.com/cdot-network/ics/pull/22/files#diff-0fd33d4fad07d7901a8c918c1b2ff17fR146) is 

> *送达或超时保证*是指数据包要么被送达并执行，要么可对发送者证明超时。

Do you think it is better?

Reply:
If the source means:

> ... the sender gets confirmation that the packet is either delivered & executed or the packet is timeout...

I prefer to mine. 

If it means:

> ... the packet is either delivered & executed or the sender gets confirmation that the packet is timeout...

the latest version is right.

Reply:
Yes, I think you are right. According to ics-004, there is indeed an explicit acknowledgement after the package has been delivered. Your changes will be included in the next commit.